### ISSUE TITLE
Stop using deprecated userId field in responses

### ISSUE THREAD

**Issue by mutahirqureshi** [hamir/islam-buddy#23]
https://developers.google.com/actions/identity/user-info#anonymous_user_identity

**Comment by mutahirqureshi** [comment]
probably should start using the new save data API for Actions on Google: https://developers.google.com/actions/assistant/save-data

**Comment by a27med** [comment]
Yeah we need to get back into this since it already deprecated and we have started failing badly :(
@mutahirqureshi @hamir
